internal/service/service_test.go

```diff
@@ -30,6 +33,13 @@ type TestCtx struct {
 	server *httptest.Server
 }
 
+type monkeyPatch struct {
```

Comment:
Can we rename this to be more meaningful, maybe testOverrides etc

Reply:
I feel `monkeyPatch` best represents what we are doing here. since it is analogous to monkey patching/ MonkeyPatch in python, ruby, and a couple of other dynamic languages

Reply:
I just felt that since this is in golang, many may not be familiar with the wording, so a more generic naming would be better.

Reply:
Done